config/locales/en.yml

```diff
@@ -70,6 +70,8 @@ en:
         archived: Archived
         deactivate: Deactivate
       form:
+        add_volunteers: add volunteers
```

Comment:
Since there are suggested changes, otherwise that is pure NIT picking, it could be `Add volunteers` and the next `Edit volunteers`